### ISSUE TITLE
cli: monitor tests

### ISSUE THREAD

**Issue by webern** [bottlerocket-os/bottlerocket-test-system#43]
Support monitoring a Sonobuoy test run with the CLI. We should be to see if it is running and, when done, see the short-form results readout. (With a pre-existing EKS cluster and an EC2 Resource Provider.)

The assumption here is some sort of `testsys run --wait` flag. I think this issue represents the blocking `--wait` workflow. I'm not sure a separate issue already exists for `testsys run` followed by `testsys status [identifier]` but we will need that also. Maybe implement it for this issue or else open one.

- depends on #42
- depends on #38

`id: cli_monitor_sonobuoy`
